task/06-conditions-n-loops-tasks.js

```diff
@@ -431,7 +612,30 @@ function getMatrixProduct(m1, m2) {
  *
  */
 function evaluateTicTacToePosition(position) {
-    throw new Error('Not implemented');
+	//  throw new Error('Not implemented');
+	const check = (a, b, c) => {
+		return (a === b && b === c && a != undefined) ? true : false;
```

Comment:
"operator !== is always preferable
sometimes ''!=' may cause an error
(check at console :
""0"" != 0
and ""0"" !== 0)"

Reply:
fixed